client/src/pages/DirectoryPage/DirectoryGroups.js

```diff
@@ -14,223 +14,196 @@ import picture1 from "../../assets/images/card_small1.png";
+    }
+  }
 
-      <div className="CommunityPage_SortByButton">
-        <div className="CommunityPage_SortByText">Sort By: Location </div>
-      </div>
-      <div className="CommunityPage_body">
-        <div>
-          <header className="CommunityPage__header">Group Location:</header>
-        </div>
-        <div className="CommunityPage__group-location-container">
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"purple"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />{" "}
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />{" "}
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
+  componentDidMount() {
+    axios.get(`${BASE_URL}/groups`)
+      .then(response => {
+        this.setState({ 
+          groups: response.data
+        })
+      })
+      .catch(error => {
+        console.log('Error while fetching and parsing API request', error);
```

Comment:
Can you render an error, rather than console logging it? The users wont see this in the console, so you need to show them some error message in the interface